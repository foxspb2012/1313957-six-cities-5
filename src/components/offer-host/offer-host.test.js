import React from 'react';
import OfferHost from './offer-host';
import renderer from 'react-test-renderer';

const HOST = {
  'name': `Host`,
  'avatar': `img/avatar-angelina.jpg`,
  'isPro': false,
  'id': 1
};
const DESCRIPTION = `An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.`;

it(`Should OfferHost render correctly`, () => {
  const tree = renderer
    .create(
        <OfferHost
          host={HOST}
          description={DESCRIPTION}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
