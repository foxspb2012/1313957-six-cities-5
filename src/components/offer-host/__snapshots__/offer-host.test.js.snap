// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferHost render correctly 1`] = `
<div
  className="property__host"
>
  <h2
    className="property__host-title"
  >
    Meet the host
  </h2>
  <div
    className="property__host-user user"
  >
    <div
      className="property__avatar-wrapper  user__avatar-wrapper "
    >
      <img
        alt="Host avatar"
        className="property__avatar user__avatar"
        height="74"
        src="img/avatar-angelina.jpg"
        width="74"
      />
    </div>
    <span
      className="property__user-name"
    >
      Host
    </span>
  </div>
  <div
    className="property__description"
  >
    <p
      className="property__text"
    >
      An independent House, strategically located between Rembrand Square and National Opera, but where the bustle of the city comes to rest in this alley flowery and colorful.
    </p>
  </div>
</div>
`;
